content/index.md

```diff
@@ -1,31 +1,4 @@
 ---
 metaTitle: CRM für Kinos
 layout: HomePage
```

Comment:
you should move the content (so far just the iframe url) into this file and then get the content in the other file.

Reply:
👍